src/themes/default/modules/nag.variables

```diff
@@ -11,7 +11,7 @@
 @zIndex: 999;
 @margin: 0;
 
-@background: #777777;
+@background: #909090;
```

Comment:
Both bright (white) and dark (black) texts has enough contrast on new background??
<img width="305" alt="image" src="https://user-images.githubusercontent.com/127635/201442549-2bc33441-64d3-4ecf-b0e9-bc0d37940bee.png">

Reply:
> Do both bright (white) and dark (black) texts have enough contrast on the new bg-color?
> 
> Below is a current screenshot taken from [fomantic/Fomantic-UI-Docs#400](https://github.com/fomantic/Fomantic-UI-Docs/issues/400) and dark text is a bit low-contrast to me. <img alt="image" width="305" src="https://user-images.githubusercontent.com/127635/201442549-2bc33441-64d3-4ecf-b0e9-bc0d37940bee.png">

That's why i wanted to get this a bit better. It's still not perfect, but the whole default theme would need a groundup refactoring regarding a11y contrast colors (not only nag)

|Old|New|
|-|-|
|![image](https://user-images.githubusercontent.com/18379884/201486410-087bd1b8-618c-442a-8255-0f9f53bd34ee.png)|![image](https://user-images.githubusercontent.com/18379884/201486438-fe4fc487-4479-4e36-8063-1f8a925d47dc.png)|